runners/core-java/src/main/java/org/apache/beam/runners/core/LateDataDroppingDoFnRunner.java

```diff
@@ -149,15 +149,13 @@ public LateDataFilter(
         }
       }
 
-      Iterable<WindowedValue<InputT>> nonLateElements =
-          StreamSupport.stream(concatElements.spliterator(), false)
-              .filter(
-                  input -> {
-                    BoundedWindow window = Iterables.getOnlyElement(input.getWindows());
-                    return !canDropDueToExpiredWindow(window);
-                  })
-              .collect(Collectors.toList());
-      return nonLateElements;
+      return StreamSupport.stream(concatElements.spliterator(), false)
```

Comment:
I think the variable name served documentation value here; if we want to remove it we should add a comment.